Dockerfile

```diff
@@ -1,31 +1,40 @@
-FROM node:14-slim AS site-build
+## Build Frontend
+FROM --platform=$BUILDPLATFORM node:14-slim AS site-build
```

Comment:
The `--platform=$BUILDPLATFORM` tells the build process to utilize the host platform, instead of emulating the target platform. This is great for build steps like these that are not architecture dependent.